ambry-api/src/main/java/com.github.ambry/server/StatsHeader.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.server;
+
+import java.util.List;
+
+
+/**
+ * A model object that contains metadata information about some reported stats. For example, the kind of stats that is
+ * being reported, timestamp and etc.
+ */
+public class StatsHeader {
```

Comment:
does the class and `enum` need to be public (v/s package private)? (is this for the jackson object mapper?)